locales/de/subtitles/lucien-ducastel/lucien-ducastel-01.yml

```diff
@@ -122,22 +122,22 @@ de:
           s: 22
           from: 00:02:20,22
           to: 00:02:25,00
-          text: The strikes of 1936 were the beginning of it all.
+          text: Mit den Streiks 1936 fing alles an.
         sequence-23:
           s: 23
           from: 00:02:25,00
           to: 00:02:33,27
-          text: There was a labour movement that manifested itself. I was already
-            part of the working class.
+          text: Daraufhin bildete sich die Arbeiterbeweung heraus. Zu dieser Zeit war ich
+          bereits Teil des Arbeiterumfelds.
         sequence-24:
           s: 24
           from: 00:02:33,27
           to: 00:02:42,26
-          text: In these conditions the 1936 movement manifested itself a little bit
-            everywhere and there were big demonstrations.
+          text: Unter diesen Umständen stieg die Bewegung von 1936 überall
+           ein wenig weiter auf und es gab große Demonstrationen.
         sequence-25:
           s: 25
           from: 00:02:42,26
           to: 00:02:48,12
-          text: It meant very much to the working class movement in France. It was
-            important.
+          text: Alles, was das für die Arbeiterbewegung in Frankreich bedeutete.
```

Comment:
and continuing: `und was das alles für die Arbeiterbewegung in Frankreich bedeutete, war sehr wichtig. Das ist gewiss.`